.githooks/pre-commit

```diff
@@ -1,22 +1,38 @@
 #!/bin/bash
 cd "$(git rev-parse --show-toplevel)"
+# Pylint Path is Micky's local path, feel free to change in PR 
+PYLINT = "c:\users\micky\appdata\local\programs\python\python310\lib\site-packages" 
 ESLINT="client/node_modules/.bin/eslint"
 pwd
 
-staged=`git diff --name-only --cached --diff-filter=A --diff-filter=M | cat`
-pylint="c:\users\micky\appdata\local\programs\python\python310\lib\site-packages  --rcfile .pylintrc --load-plugins pylint_django --load-plugins pylint_quotes --reports=no"
+if [[ ! -x "$PYLINT" ]]; then
+	  printf "\t\033[41mPlease install Pylint\033[0m (pip install pylint)\n"
+	    exit 1
+    fi
+
+    STAGED_FILES=($(git diff --cached --name-only --diff-filter=ACM | grep ".py\{0,1\}$"))
+
+    echo "Checking ${#STAGED_FILES[@]} staged files for Pylint Errors..."
+
+    if [[ "$STAGED_FILES" = "" ]]; then
+	      exit 0
+      fi
+
+      $PYLINT "${STAGED_FILES[@]}" --fix
 
-if [ "$staged" ]
-then
-  target=`cat "$staged" | grep '.py$'`
-else
-  target=`git diff --name-only | cat | grep '.py'`
-fi
+      PYLINT_EXIT="$?"
 
-if [ "$target" ]
-then
-  $pylint "$target"
-fi
+      # Re-add files since they may have been fixed
+      git add "${STAGED_FILES[@]}"
+
+      if [[ "${PYLINT_EXIT}" == 0 ]]; then
+	        printf "\n\033[42mCOMMIT SUCCEEDED\033[0m\n"
```

Comment:
Maybe change commit succeeded to something like "pylint passed" since the ESlint test comes after still